deploy/02_role.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Role
```

Comment:
If you are changing these values you need to regenerate the bundle.

Reply:
https://github.com/openshift/kueue-operator?tab=readme-ov-file#operator-bundle-development